roles/common/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
 - name: Get users
   getent:
     database: passwd
+ # fix this after Ansible 2.6
```

Comment:
Any chance we can elaborate on exactly how we should fix it and what needs to be done after Ansible 2.6? Maybe @ripleymj can provide some insight here?

Reply:
It should be changed to `loop: "{{ getent_passwd|dict2items }}"` once mint uses Ansible 2.6
[as seen here.](https://docs.ansible.com/ansible/2.6/user_guide/playbooks_loops.html#with-dict)

Ansible 2.5 just doesn't support this type of styling yet.

Reply:
^^this

Reply:
Nice! Thanks!

Reply:
```suggestion
 # fix this after Ansible 2.6, use loop: "{{ getent_passwd|dict2items }}"
```